package.json

```diff
@@ -67,7 +67,7 @@
     "react-dom": "17.0.2",
     "react-dropzone": "^11.3.2",
     "react-helmet": "6.1.0",
-    "react-hook-form": "^6.15.8",
+    "react-hook-form": "^7.8.7",
```

Comment:
Was this required or just a housekeeping step ? Anything that broke or is of interest for us ? Did this cause the update in the contact form (breaking changes?)

Reply:
Yes, the other were already updated. It broke the `register`-function of the input-fields. (The name should be added to the function itself, not only optional validation and error-messages as before)
For reference [https://react-hook-form.com/api/useform/register](https://react-hook-form.com/api/useform/register)